src/app/register/register.module.ts

```diff
@@ -0,0 +1,20 @@
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { RegisterRoutingModule } from './register-routing.module';
+import { RegisterComponent } from './register.component';
+import { HttpClientModule } from '@angular/common/http';
+import { SharedModule } from '../shared/shared.module';
+
+
+@NgModule({
+  declarations: [RegisterComponent],
+  imports: [
+    CommonModule,
+    RegisterRoutingModule,
```

Comment:
Register and Login doesn't need to exist as two separate modules. Module grouping is done around the commonality of the work performed. Login and Register are both about user's authentication management and thus could be in one `auth` module.

Reply:
done